### ISSUE TITLE
Cacti1.2 installation

### ISSUE THREAD

**Issue by tyit** [Cacti/cacti#2189]
The new version is installed complexly, is there any script-like automatic installation or package optimized ISO image?

Thank you

**Comment by netniV** [comment]
Can you provide the following information:

- OS & Version
- PHP Version
- Cacti Version

**Comment by tyit** [comment]
- CentOS Linux release 7.4.1708 (Core)
- Nginx-1.14.1
- Php7.0
- Mariadb5.5
- Cacti  1.2.0 Beta 4

**Comment by netniV** [comment]
For CentOS, @mortenstevens provides packages to install Cacti.  He may not have built the Beta 4 one yet as that's just been released.

**Comment by mortenstevens** [comment]
@netniV 
@tyit 

We are shipping only stable cacti releases for Fedora and RHEL (Centos) via Fedora-EPEL. https://fedoraproject.org/wiki/EPEL

**Comment by netniV** [comment]
Ah, that would explain things.  It's a shame there's no "unstable" version where people can test betas.

**Comment by cigamit** [comment]
We just need to release 1.2.  I would say if we find nothing 'severe' in the next week, we should push @ronytomen to push out the release.

**Comment by netniV** [comment]
That is true but going forward there will always be people who want to use the latest greatest before it's declared stable.  Debian has the concept of stable vs unstable hence my use of that terminology but effectively it's the same thing.  Stable Packages tend to be a release or two behind (unless there's been no update).
